apps/Standalone/src/designer/app/AzureLogicAppsDesigner/Queries/FetchHybridApps.ts

```diff
@@ -10,7 +10,7 @@ export const useFetchHybridApps = () => {
       if (!environment.armToken) {
         return [];
       }
-      const query = `resources | where type =~ "microsoft.app/containerApps" and kind contains "workflowapp"`;
```

Comment:
when pushing to main, we should remove this line; we'll operate under the assumption that the ARG change will be merged soon

Reply:
reverted